pages/app/index.ftl

```diff
@@ -2,8 +2,12 @@
 
 <@tmpl.page current="test-app" title="Test application" noindex=true nocsp=true>
 
-<script src="keycloak.js" type="text/javascript"></script>
-<script src="app.js" type="text/javascript"></script>
+<#if useLegacyKeycloakJs>
+    <script src="keycloak.js" type="text/javascript"></script>
+    <script src="app.js" type="text/javascript"></script>
```

Comment:
Maybe `app-legacy.js` for the old approach, and update `app.js` to the new approach, but again doesn't matter as we will just make it the new approach only after the release.